specifications/api/swagger.yaml

```diff
@@ -2407,6 +2407,23 @@ components:
       properties:
         transaction: *serialisedTransactionEncoded
 
+    ApiMintedBurnedInfo: &ApiMintedBurnedInfo
+      type: object
+      required:
+        - verification_key_index
+        - policy_id
+        - asset_name
+        - subject
+        - policy_script
```

Comment:
I guess that `policy_id`, `subject` and `policy_script` should not be there based on the rendered payload?

![Screenshot from 2022-02-23 16-52-32](https://user-images.githubusercontent.com/42900201/155355987-9374eec6-0aab-4825-b5f7-c8a766eda4f1.png)